test/common/testlib.py

```diff
@@ -46,6 +46,8 @@
 
 TEST_DIR = os.path.normpath(os.path.dirname(os.path.realpath(os.path.join(__file__, ".."))))
 BOTS_DIR = os.path.normpath(os.path.join(TEST_DIR, "..", "bots"))
+# ENV  variable name for skipping tests based on generic rule like TEST_SKIP="tag1 tag2"
+TEST_SKIP_ENV_VARIABLE = "TEST_SKIP"
```

Comment:
I find it confusing to put the name of a variable into another variable. I think it's fine to just put it straight into `skipGeneric()`.